samples/camera/camera.rst

```diff
@@ -0,0 +1,10 @@
+.. _camera-samples:
+
+Camera Samples
+##############
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   **/*
```

Comment:
This assumes there will be more than one camera sample, and all these samples are in subdirectories.  In this PR though, the (one) sample is in the same directory as this camera.rst  so the reference to ``**/*`` is not finding any rst files and showing an error message.

Reply:
Remove the camera.rst and describe the sample in README.rst.